packages/manager/modules/pci/src/projects/project/storages/databases/add/add.scss

```diff
@@ -1,3 +1,25 @@
 .order-summary {
   z-index: 1009;
 }
+
+tr.table-single-select-row {
+  cursor: pointer;
+}
+tr.table-single-select-row:hover {
+  background-color: #bef1ff;
+}
+.table-selected {
+  background-color: #def8ff;
+}
```

Comment:
**question:** Could you please avoid using custom colors? You can check and use the appropriate background from the classes in `ui-kit`.
Reference: https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-helpers-backgrounds--primary

Reply:
Actually, this css is not used anymore, I removed it here: https://github.com/ovh/manager/pull/8392/commits/913ab877b1b4954f336b29d90f2d1f0946e00f00.